extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -89,6 +100,27 @@ export class AmpExperiment extends AMP.BaseElement {
       return experiments;
     });
   }
+
+  /**
+   * Parses URL fragment and returns variant override.
+   * @return {!Object<string, string>}
+   * @private
+   */
+  getVariantOverride_() {
+    const queries = parseQueryString(
```

Comment:
@cramforce could you point me to where the fragment can be removed? I was following the way how development=1 was parsed by mode.js, why it cannot apply to this case?